### DISCUSSION TITLE
weight link get 404 error, may I know if there is other way to get pre-trained weights?

### DISCUSSION THREAD

**Discussion by spencer03172023** [Project-MONAI/tutorials#1339] [Q&A]
**Please use MONAI's Discussions tab**
For questions relating to MONAI usage, please do not create an issue.

Instead, use [MONAI's GitHub Discussions tab](https://github.com/Project-MONAI/MONAI/discussions). This can be found on the MONAI repository's main page (not the tutorials) next to Issues and Pull Requests along the top.

**Comment by KumoLiu** [comment]
Hi @spencer03172023, could you please give more information about what pre-trained weights do you want get? Thanks!
